core/src/main/scala/sttp/tapir/RangeValue.scala

```diff
@@ -0,0 +1,26 @@
+package sttp.tapir
+
+import scala.util.Try
+
+case class RangeValue(unit: String, start: Int, end: Int) {
```

Comment:
this models the `Content-Range` header value, right? Maybe we should move this to `sttp.model.headers`, along with the `Range` header value (and parsing/serialising with the usual rules)